qemu/tests/qemu_img.py

```diff
@@ -331,7 +331,10 @@ def info_test(cmd):
         if image_format not in img_info:
             test.fail("Got unexpected format of image '%s'"
                       " in info test" % image_name)
-        if image_size not in img_info:
+
+        image_si = re.sub("\\D", "", image_size)
```

Comment:
this verification is crude, here is what I believe a better approach:
1. use `QemuImg.info` with `output='jsoin'` to retrieve the size in bytes.
2. convert the size specified in params into bytes.
3. do the comparison.